bears/r/FormatRBear.py

```diff
@@ -1,22 +1,69 @@
 from coalib.bearlib.abstractions.Linter import linter
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
 from coalib.parsing.StringProcessing import escape
 
 
+def _map_to_r_bool(py_bool):
+    return 'TRUE' if py_bool else 'FALSE'
+
+
 @linter(executable='Rscript',
         output_format='corrected',
         prerequisite_check_command=('Rscript', '-e', 'library(formatR)'),
         prerequisite_check_fail_message='Please install formatR for this bear '
                                         'to work.')
 class FormatRBear:
-
     """
     Check and correct formatting of R Code using known formatR utility.
     """
     LANGUAGES = "R"
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
+    def create_arguments(filename, file, config_file,
+                         r_keep_comments: bool=True,
+                         r_keep_blank_lines: bool=True,
+                         r_braces_on_next_line: bool=False,
+                         r_use_arrows: bool=False,
+                         tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
+                         r_max_expression_length: int=20):
+        """
+        :param r_keep_comments:         Determines whether comments are
+                                        kept or not.
+        :param r_keep_blank_lines:      Determines whether blank lines
+                                        are kept or not.
+        :param r_braces_on_next_line:   Determines wheter a brace should
+                                        be placed on the next line
+                                        Example: if `True`,
+                                        "if (...)
+                                        {"   instead of "if (...) {"
+        :param r_use_arrows:            Determines if either the assign
+                                        operator ``=`` or the arrow ``<-``
+                                        should be used
+                                        Example: if `True`,
+                                        "x = 1" becomes "x <- 1".
+        :param tab_width:               Number of space for indentation.
+        :param r_max_expression_length: Maximum number of characters for an
+                                        expression.
+                                        Example: if 20 then
+                                        "1 + 1 + 1 + 1 + 1 + 1 +
+                                             1" is used instead of
+                                        "1 + 1 + 1 + 1 + 1 + 1 + 1"
```

Comment:
also double backticks for code expressions.
And having here the "new" documentation style would simplify reading as well :+1: